### DISCUSSION TITLE
All Issues/PRs Across All Repos

### DISCUSSION THREAD

**Discussion by NTCoding** [ddd-crew/ddd-starter-modelling-process#52] [Team Posts]
Is there a way we can view all open issues and PRs across all ddd-crew github repositories?

**Comment by NTCoding** [comment]
Think this is the answer:

<img width="785" alt="Screenshot 2020-06-26 at 08 31 34" src="https://user-images.githubusercontent.com/692094/85832211-8ea70580-b787-11ea-8195-c46852b88142.png">

**Comment by yellowbrickc** [comment]
Let's bookmark the link: 

[all open issues](https://github.com/issues?q=user%3Addd-crew+is%3Aopen)
